src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -763,6 +763,78 @@ private void DecodeDllImportAttribute(ref DecodeWellKnownAttributeArguments<Attr
                     preserveSig);
             }
         }
+
+        private void DecodeModuleInitializerAttribute(DecodeWellKnownAttributeArguments<AttributeSyntax, CSharpAttributeData, AttributeLocation> arguments)
+        {
+            Debug.Assert(arguments.AttributeSyntaxOpt is object);
+
+            if (MethodKind != MethodKind.Ordinary)
+            {
+                // Ignore cases where there will already be a warning due to the AttributeTargets.Method usage
```

Comment:
```suggestion
                // Ignore cases where there will already be an error due to the AttributeTargets.Method usage
```
AFAIK AttributeUsage violations are always errors.

Reply:
Yes, misspoke.